lib/src/switch/Switch.stories.tsx

```diff
@@ -0,0 +1,154 @@
+    <ExampleContainer>
+      <Title title="With label after" theme="light" level={4} />
+      <DxcSwitch label="Switch" labelPosition="after" />
+    </ExampleContainer>
+    <ExampleContainer>
+      <Title title="Without label" theme="light" level={4} />
+      <DxcSwitch />
+    </ExampleContainer>
+    <ExampleContainer>
+      <Title title="With large label" theme="light" level={4} />
+      <DxcSwitch label="Very long long long label or huge as you want to call it" size="medium" />
+    </ExampleContainer>
+    <ExampleContainer>
+      <Title title="With long word" theme="light" level={4} />
+      <DxcSwitch
+        label="Large texttttttttttttttttttttttttttttttttttttttttttttttttttttttttttttt"
+        labelPosition="after"
+        size="medium"
+      />
+    </ExampleContainer>
+    <ExampleContainer>
+      <Title title="Checked" theme="light" level={4} />
+      <DxcSwitch label="Switch" checked />
+    </ExampleContainer>
+    <ExampleContainer>
+      <Title title="Required" theme="light" level={4} />
+      <DxcSwitch label="Switch" required />
+    </ExampleContainer>
+    <ExampleContainer>
+      <Title title="Disabled" theme="light" level={4} />
+      <DxcSwitch label="Switch" disabled />
+    </ExampleContainer>
+    <ExampleContainer>
+      <Title title="Disabled required" theme="light" level={4} />
+      <DxcSwitch label="Switch" disabled required labelPosition="after" />
+    </ExampleContainer>
+    <ExampleContainer>
+      <Title title="Disabled checked" theme="light" level={4} />
+      <DxcSwitch label="Switch" disabled checked labelPosition="after" />
+    </ExampleContainer>
+    <BackgroundColorProvider color="#333333">
+      <DarkContainer>
+        <ExampleContainer>
+          <Title title="With label" theme="dark" level={4} />
+          <DxcSwitch label="Switch" />
+        </ExampleContainer>
+        <ExampleContainer>
+          <Title title="Checked" theme="dark" level={4} />
+          <DxcSwitch label="Switch" checked />
+        </ExampleContainer>
+        <ExampleContainer>
+          <Title title="Required" theme="dark" level={4} />
+          <DxcSwitch label="Switch" required />
+        </ExampleContainer>
+        <ExampleContainer>
+          <Title title="Disabled" theme="dark" level={4} />
+          <DxcSwitch label="Switch" disabled />
+        </ExampleContainer>
+        <ExampleContainer>
+          <Title title="Disabled required" theme="dark" level={4} />
+          <DxcSwitch label="Switch" disabled required labelPosition="after" />
+        </ExampleContainer>
+        <ExampleContainer>
+          <Title title="Disabled checked" theme="dark" level={4} />
+          <DxcSwitch label="Switch" disabled checked labelPosition="after" />
+        </ExampleContainer>
+        <ExampleContainer pseudoState="pseudo-focus">
+          <Title title="Focused" theme="light" level={4} />
+          <DxcSwitch label="Switch" />
+        </ExampleContainer>
+      </DarkContainer>
```

Comment:
Here the pseudoState is not doing anything as we are overriding the focus styles from material. We can use the focus function like this: (both for light and dark tests)

```
const Switch = () => (
  <ExampleContainer>
    <Title title="Focused" theme="light" level={4} />
    <DxcSwitch label="Switch" />
  </ExampleContainer>
);

export const FocusedSwitch = Switch.bind({});
FocusedSwitch.play = async ({ canvasElement }) => {
  const canvas = within(canvasElement);
  canvas.getByRole("checkbox").focus();
};
```

Reply:
This was a residual example I forgot to remove. I will add a new story for dark focused Switch!